src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -85,6 +85,21 @@ public SortClause deserialize(JsonParser parser, DeserializationContext ctxt)
           sortExpressions.add(exp);
           break;
         }
+      } else if (DocumentConstants.Fields.VECTOR_EMBEDDING_TEXT_FIELD.equals(path)) {
+        if (totalFields > 1) {
```

Comment:
Maybe add a comment indicating logic here (vector embedding update must be the only clause).

Reply:
Added comment `Vector search can't be used with other sort clause`